### ISSUE TITLE
Notification on Garmin no longer shows only the new message when conversation history is enabled

### ISSUE THREAD

**Issue by WereGoingOcean** [klinker-apps/pulse-sms-issues#1031]
## Confirm this Issue

- [X] I have [searched](https://github.com/klinker-apps/messenger-issues/issues), by keyword, for my problem to verify that there aren't any open or closed issues around this behavior.
- [X] I have [searched the help page](https://messenger.klinkerapps.com/help) to verify that it does not have any topics relevant to this issue.
- [X] I have verified that this is something that is reproducible.
- [X] This is not a general support questions. Support should be done over [email](mailto:luke@klinkerapps.com).

## Device/Platform Information

**Device with issue:** OnePlus 6t on Android 10 + Garmin Forerunner 645
**Primary device:** OnePlus 6t on Android 10
**App version:** Ex: v5.4.5

## Describe the Bug

I've always used the "Show Conversation History" notification option and not had an issue with notifications showing on my watch. Recently though instead of showing just the newest message it concatenates all the messages from the sender that appear in the history and displays them, usually hiding the actual new message due to length.

### Steps to Reproduce

1. Enable "Show Conversation History" in the notification settings.
2. Receive a message in a conversation with a few existing messages.
3. See that watch notification no longer displays just the new message.

### Expected Behavior

The notification that appears on the watch would only display the newest message received.

### Additional Information

To explain with an example:

If the convo history looked like this
| Left | Right |
| --- | --- |
| Let me know what I owe you for the car. | |
| | What did they charge you when we picked it up? |
| 48 | |
| | Looks like 500 then. |

and the left side sent a new message "Alright I'll venmo you" what actually displays on the watch notification would be something like:

Let me know what I owe 
you for the car.
48

**Comment by klinker24** [comment]
Yeah I don't know why this is happening. I am seeing it myself, but there haven't been any Pulse changes for a few weeks. I believe that this is something Garmin Connect changed and will need to fix. All that I can do is post notifications and Garmin sends the content to the watch.

Even when "show conversation history" is turned off, this happens.

**Comment by the38steps** [comment]
Can confirm this is also happening with Samsung Galaxy Watch3 synced with Samsung S20 Ultra.
